torch/_inductor/kernel/mm.py

```diff
@@ -286,3 +287,25 @@ def tuned_mixed_mm(mat1, mat2, mat2_dtype):
             **mm_options(config, k, layout, b_prologue_cast_type),
         )
     return autotune_select_algorithm("mixed_mm", choices, [mat1, mat2], layout)
+
+
+def tuned_fused_int_mm_mul(mat1, mat2, mat3, out_dtype, *, layout=None):
```

Comment:
Add a comment explaining why there is no aten option here, and that this is only enabled by `config.force_fuse_int_mm_with_mul`